### PR TITLE
Corrected the docstrings and added seasons extraction code

### PR DESCRIPTION
**Pull request by Manusreekumar**
Closes: #67

### CODE REVIEW

**Comment by Manusreekumar** [comment]
@gsganden Will this PR require any further changes ?

**Comment by gsganden** [comment]
Sorry, I haven't had a chance to review it yet. I will get to it when I can.

File: autofocus/build_dataset/lpz_2016_2017/ops.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def record_is_grayscale(
 
     Side effect
     -----------
-    Adds a "mean_brightness" items to log_dict[inpath]
+    Adds a "grayscale" items to log_dict[inpath]
```
Comment (gsganden) [inline autofocus/build_dataset/lpz_2016_2017/ops.py]: Please change "items" to "item."

File: autofocus/build_dataset/lpz_2016_2017/ops.py [hunk 2]
```diff
@@ -42,11 +42,13 @@ def record_mean_brightness(
     image: np.array, inpath: PathOrStr, log_dict: DefaultDict[str, dict]
 ) -> np.array:
     """
-    Record whether image is grayscale.
+    Record the mean brightness of image.
 
     In this dataset, grayscale images have been saved as three-channel
```
Comment (gsganden) [inline autofocus/build_dataset/lpz_2016_2017/ops.py]: We can delete this note about grayscale images, which isn't really relevant to this function. In fact, I think we can just delete this paragraph of the docstring.

**Pull request merged**